### DISCUSSION TITLE
Animate Tool - "jumping" fuzzy movement

### DISCUSSION THREAD

**Discussion by MiniArmorer** [opentoonz/opentoonz#6399] [Q&A]
Hi!
I started working with OpenToonz and I have question.
When I try to animate movement with Animate Tool, moving object seems to be blurred and it causes uncomfortable effect.
There are "streaks" and movement seems not ot be smooth.
What am I doing wrong? Is it my fault or monitor setting? Or I need to change sth in program settings? Too many frames or too low?
I spend plenty of hours to fix that and I can't find solution.
I'll be really gratefull for any help :)

https://user-images.githubusercontent.com/86148978/122638337-d68b0f00-d0f3-11eb-98d8-5e4b71b9a6a0.mp4

**Comment by RodneyBaker** [comment]
Converting this to Discussion so others can explore the provided information.
The age of the report suggests this may no longer be an issue.
